src/Model/Contact.php

```diff
@@ -3068,4 +3055,228 @@ static public function getSuggestions(int $uid, int $start = 0, int $limit = 80)
 
 		return array_slice($contacts, $start, $limit);
 	}
+
+	/**
+	 * Add public contacts from an array
+	 *
+	 * @param array $urls
+	 * @return array result "count", "added" and "updated"
+	 */
+	public static function addContactsByArray(array $urls)
```

Comment:
Please rename this method:
```suggestion
	public static function addByUrls(array $urls)
```